manual/source/datarules.rst

```diff
@@ -757,21 +765,21 @@ plottable data is as follows:
 #. Plot the *signal* data, given *axes* and *AXISNAME_indices*.
 
 
-When all the ``default`` and ``signal`` attributes are present, this 
-Python code will identify directly the default plottable data 
+When all the ``default`` and ``signal`` attributes are present, this
+Python code will identify directly the default plottable data
 (assuming a ``plot()`` function has been defined by some code::
 
     root = h5py.File(hdf5_file_name, "r")
-    
+
     default_nxentry_group_name = root.attrs["default"]
     nxentry = root[default_nxentry_group_name]
-    
+
     default_nxdata_group_name = nxentry.attrs["default"]
```

Comment:
technically an extra step is required in case there is an NXsubentry present.

Reply:
I agree this is a good thing for a new issue.

Reply:
Added a `TODO` comment to the example code.

I we are to loop through a longer `default` chain, we should factor that into the other base classes.

Reply:
adjusted the wording